### ISSUE TITLE
Add a hide facilities to the facility list of users in User tab

### ISSUE THREAD

**Issue by Pragati1610** [coronasafe/care_fe#4249]
Issue
* Currently to link a new facility we need to scroll a lot before we can view link a new facility option.
* A new section of viewing skills of users is being added in https://github.com/coronasafe/care_fe/issues/4248 . Having 2 lists would make it difficult to view and scroll again

Solution
* A simpler UI for viewing the 2 sections

Additional feature
* It would be easier to have a hide facility list for the same

**Comment by rithviknishad** [comment]
@Pragati1610 duplicate of https://github.com/coronasafe/care_fe/issues/4116
